notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "linelist_table = astropy.table.Table()\n",
+    "linelist_table['linename'] = ['Halpha', 'Hbeta']\n",
+    "linelist_table['rest'] = [6563, 4862]*u.angstrom\n",
+    "linelist_table['more information'] = ...\n",
+    "\n",
+    "specviz.load_linelist(linelist_table)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, either a plugin or a pane specifically dedicated to this (perhaps associated with the \"redshift\" slider?) should present a few options: a \"galaxies\" linelist, a \"stars\" linelist, and a \"load file...\" option.  The first two would be curated lists (ideally in `specutils` but could be in `jdaviz`), while \"load file...\" would prompt the user for a file, which would then be read as `astropy.table.Table.read('/path/to/file')`.  This ensures the same line list tables can be used in the notebook and the UI setting."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Regardless of the above, in the end vertical lines should appear at the locations specified in `'rest'`, and the UI should have at least a few styling options exposed (although that may be a lower-priority stretch goal in detail)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SPLOT commands related to redshifting operations addressed above"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### u -- Adjust  the user coordinate scale.\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Adjust  the user coordinate scale.  There are three options, 'd' mark a position with the  cursor  and  doppler  shift  it  to  a specified  value,  'z'  mark  a  position  with  the  cursor and zeropoint shift it  to  a  specified  value,  or  'l'  mark  two postions   and   enter   two  values  to  define  a  linear  (in  wavelength) dispersion scale.  The  units  used  for  input  are those  currently  displayed.   A  wavelength  scale set this way will  be  used  for  other  spectra  which  are  not  dispersion  corrected."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### v -- Toggle  to  a velocity scale using the position of the cursor as         the velocity origin and back.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Functionality for multidimensional spectra\n",
+    "\n",
+    "SPLOT includes functionality (partly from broader IRAF) for doing spectral operations on multidimensional arrays like \"2d\" spectra or data cubes.  In general this is not a target for `specviz` so it is not a priority of the UI, but much of the SPLOT workflow can be addressed from the notebook by slicing spectrum objects:"
+   ]
+  },
```

Comment:
Does this point out a gap in our current development plan for viz tools?   What viewer do I use if I want to inspect 2D spectra?  Should a stretch goal for SpecViz be to launch a second viewer pane for the 2D spectrum?  Use case: I have a 2D spectrum and an associated 1D spectrum that I want to clean by specifying bad pixels in the 2D version.

Reply:
Another use case: I have a 2d spectrum and want to generate new 1d spectra to compare with the default associated 1d spectrum.  

Some of this functionality should be in cubeviz as well.  Our users did want some form of this in the UI but I think it could be a stretch goal.

Reply:
This is explicitly a goal for mosviz, *not* specviz.  In the mosviz concept notebook I outlined the idea that depending on how you load data you can get *only* the 2d and 1d spectrum.  The idea I mean here is that the 1d viewer in `mosviz` *is* `specviz`, and I'm just trying to say "even though it's in SPLOT, these functions map onto mosviz or cubeviz, not specviz".  I'll try to reword that more carefully.

Reply:
Oh and re: @havok2063's point, there might be a "branding" concern that the "multi object spectroscopy" viewer also supports a *single* 2d + 1d spectrum.  It might be we should consider having there be an intentionally stripped-down viewer that is just those parts of Mosviz? We don't want to re-implement any of it if we can avoid it, but it might be just to lower user confusion it's worth doing.

Either way I think it doesn't belong in this notebook per se, but maybe needs a separate issue?

Reply:
Yeah I think you're going to want something more than specviz, more than whatever future imviz there might be; and something less than mosviz.   I think it's worth creating several pre-built configurations that cover 95% of peoples' use cases.  The rest can be through config customization which you've essentially already enabled.   

On a related note, similar to your recent prioritization of specviz, I wonder if the next prioritization ought to be imviz, as that's another core viewer IMM.   Everything else is some combination of the two.  I regard mosviz as 1(2?) imviz+1 specviz+table and cubeviz as 3 imviz+1 specviz.  But I leave this up to y'all.